docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
 recommonmark==0.6.0
-# pin this to the last Py2 release
-Sphinx==1.8.5  # pyup: <2.0
+# Pin this to the same major version as https://docs.python.org/3/
+Sphinx==2.4.4  # pyup: <2.0
```

Comment:
Let\s remove the pyup annotation.

Reply:
We might still need this, if py3.8 still breaks in Sphinx 3 (the official Python doc site still uses Sphinx 2), but we do need to update the annotation accordingly.

Reply:
Well, turns out we do. Sphinx 3.1.1 doesn't work with Python 3.8 so we'll have to pin this to v2.